docker/ssl/nginx-image/start.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+sleep 3
```

Comment:
This makes the letsencrypt container to attempt verification before nginx is up to serve the `/.well-known` URI, thus creating a race condition.